scheduled-jobs/build/poll-payload/Jenkinsfile

```diff
@@ -4,27 +4,37 @@
 import groovy.transform.Field
 import java.io.FileNotFoundException
 import java.net.URLEncoder
+import groovy.json.*
```

Comment:
bit broad, is there a need for anything beyond `JsonOutput`?

Reply:
+1 let's import only what we need ;D